release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2112,6 +2112,9 @@ issue by deleting the Elasticsearch deployments in sequence.
 * {product-title} 4.4 is not shipping with a v.4.4 Metering Operator. Customers can install or continue to run the v.4.3 Metering Operator on {product-title} 4.4 clusters.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ1829035*])
 
+* The web console downloads deployment will fail on nodes configured with `ipv6.disable=1`.
```

Comment:
I think you need `(link:https)` instead of the `(linkhttps)` to make the link work!

Otherwise lgtm.

Reply:
Ha! Missed that typo. Nice catch. Thank you!